pywbemtools/pywbemcli/_connection_repository.py

```diff
@@ -56,11 +56,77 @@
 BAK_FILE_SUFFIX = 'bak'
 
 
-class ConnectionsFileNotFoundError(Exception):
+class ConnectionsFileError(Exception):
     """
     Exception indicating that the connections file was not found.
     """
-    pass
+    def __init__(self, msg):
+        """
+        Parameters:
+          msg (:term:`string`):
+            Message text for the exception.
+        """
+        # pylint: disable=useless-super-delegation
+        super(ConnectionsFileError, self).__init__(msg)
+
+
+class ConnectionsFileWriteError(ConnectionsFileError):
+    """
+    Exception to be used for connection file write errors.
+
+    """
+    def __init__(self, connections_file_name, message):
+        """
+        Parameters:
+          connection_file_name (:term:`string`):
+            Path of the connections file
+
+          message (:term:`string`): Error message.
+        """
+        assert message is None or isinstance(message, six.string_types), \
+            str(type(message))
+        msg = 'Connection file: "{0}" write error. {1}'.format(
+            connections_file_name, message)
+        super(ConnectionsFileWriteError, self).__init__(msg)
+
+
+class ConnectionsFileLoadError(ConnectionsFileError):
+    """
+    Exception that defines errors in loading the connection file. These
+    errors are due to invalid YAML or YAML key names that are required
+    but missing from the file being loaded.
+    Also, physical IO errors in file load..
+
+    """
+    def __init__(self, connections_file_name, message):
+        """
+        Parameters:
+          connection_file_name (:term:`string`):
+            Path of the connections file
+
+          message (:term:`string`): Error message.
+        """
+        assert message is None or isinstance(message, six.string_types), \
+            str(type(message))
+        msg = 'Connection file: "{0}"; {1}'.format(
+            connections_file_name, message)
+        super(ConnectionsFileLoadError, self).__init__(msg)
+
+
+class ConnectionsFileNotFoundError(ConnectionsFileError):
+    """
+    Parameters:
+
+        Parameters:
+          connection_file_name (:term:`string`):
+            Path of the connections file
+
+    """
+    def __init__(self, connections_file_name):
+        """
+        """
+        super(ConnectionsFileNotFoundError, self).__init__(
+            "Connection file: {0} does not exist".format(connections_file_name))
```

Comment:
Be consistent with quotes around the file name and the term used for the file, so change the message to:
```
'Connections file does not exist: "{0}"'.
format(connections_file_name)
```